src/app/quickview/quickview.test.ts

```diff
@@ -0,0 +1,121 @@
+import {
+    async,
+    ComponentFixture,
+    TestBed,
+  } from '@angular/core/testing';
+  import { By } from '@angular/platform-browser';
+  import { BasketComponent } from '../basket/basket.component';
+  import { ButtonComponent } from '../button/button.component';
```

Comment:
there are a lot of unused imports here, we should remove them if we are not using them